package.json

```diff
@@ -19,6 +19,8 @@
     "dependencies": {
         "@companieshouse/api-sdk-node": "^2.0.70",
         "@companieshouse/structured-logging-node": "^1.0.4",
+        "@companieshouse/web-security-node": "^2.0.0",
```

Comment:
Main has remove this dependency. 
`"@companieshouse/web-security-node": "^2.0.0",`
Should I remove it?